content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/reordering_rules.rst

```diff
@@ -59,25 +56,34 @@ If a :abbr:`BoM (Bill of Materials)` does not already exist for the product, sel
 :guilabel:`New` to configure a new :abbr:`BoM (Bill of Materials)`.
 
 .. image:: reordering_rules/bom-smart-button.png
-   :align: center
    :alt: The Bill of Materials smart button on a product form.
 
 Create new reordering rules
 ===========================
 
-To create a new reordering rule, navigate to :menuselection:`Inventory app --> Configuration -->
-Reordering Rules`, then click :guilabel:`New`, and fill out the new line as follows:
+To create a new reordering rule, navigate to
+:menuselection:`Inventory app --> Operations --> Replenishment`, then click :guilabel:`New`,
+and fill out the following fields for the new reordering rule line item:
 
 - :guilabel:`Product`: The product that is replenished by the rule.
-- :guilabel:`Location`: The location where the product is stored.
+- :guilabel:`On Hand`: The number of units currently available in inventory.
+- :guilabel:`Forecast`: The number of units expected to be available in inventory after all orders
+  are taken into account.
+- :guilabel:`Route`: The
+  :doc:`manufacturing route <../../shipping_receiving/daily_operations/use_routes>` for this product
```

Comment:
Suggestion to rename this to something inventory or warehouse-specific, since in some business cases manufacturing is not involved in their ops, such as with product resellers.

```suggestion
  :doc:`supply-chain route <../../shipping_receiving/daily_operations/use_routes>` for this product
```